### DISCUSSION TITLE
checkoutComplete with manual payment outside Saleor

### DISCUSSION THREAD

**Discussion by ohuu** [saleor/saleor#13574] [Q&A]
I'm having some trouble completing the checkout process. I'm using the `checkoutComplete` graphql query but the documentation says `This action requires successful payment before it can be performed`. In our case payment is actually taken manually (outside of Saleor etc) once an order is placed.

I noticed that the mutation takes a parameter named `paymentData` which is a `JSONString`. I'm not sure what data is expected here. Even if we could take payment via Stripe or PayPal etc it's unclear what to send. I tried passing an empty JSON object but (unsurprisingly) I get an error `Provided payment methods can not cover the checkout's total amount`.

How can I complete a checkout or create an order without having to take payment?

**Comment by peteror** [comment]
I had the same issue - ended up using the dummy card payment plugin to make a dummy payment and get a payment Id I could create the order with.
